frontstage/views/secure_messaging/message_get.py

```diff
@@ -28,7 +28,7 @@ def view_conversation(session, thread_id):
     try:
         refined_conversation = [refine(message) for message in reversed(conversation['messages'])]
     except KeyError as e:
-        logger.exception("Message is missing important data", thread_id=thread_id, party_id=party_id)
+        logger.error("Message is missing important data", thread_id=thread_id, party_id=party_id)
```

Comment:
these should be single quotes but we dont enforce it so 🤷‍♂️

Reply:
Might as well do it now 🤷‍♀️